sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/HiveThriftServer2Suite.scala

```diff
@@ -384,4 +388,32 @@ class HiveThriftServer2Suite extends FunSuite with Logging {
       }
     }
   }
+
+  test("SPARK-5100 monitor page") {
```

Comment:
Note that there's a UISeleniumSuite test (currently disabled) for writing much more complex web UI tests for Spark.  Probably overkill here, but just wanted to point it out.